piker/brokers/ib.py

```diff
@@ -346,9 +403,17 @@ async def _aio_get_client(
     """
     # first check cache for existing client
 
+    # breakpoint()
     try:
-        yield _client_cache[(host, port)]
-    except KeyError:
+        if port:
+            client = _client_cache[(host, port)]
```

Comment:
This actually wasn't working before and makes it so IB api clients are actually reused by multiple consuming actors; avoids client throttling that would result.